tgui/packages/tgui/interfaces/AlertComputer.tsx

```diff
@@ -0,0 +1,127 @@
+import { Button, Flex, Icon, Table } from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+interface AlertComputerData {
+  alerts: AlertData[];
+}
+
+interface AlertData {
+  zone: string;
+  area_ckey: string;
+  atmos: number;
+  fire: number;
+  power: number;
+  motion: number;
+}
+
+export const AlertComputer = () => {
+  const { data } = useBackend<AlertComputerData>();
+  const { alerts } = data;
+  return (
+    <Window title="Current Alerts" width={600} height={420}>
+      <Window.Content scrollable>
+        <Table>
+          <Table.Row header fontSize="1.1em" className="candystripe">
+            <Table.Cell header width="200px">
```

Comment:
```suggestion
            <Table.Cell header>
```
Avoid hard-coding the width of at least one column, so if someone makes the container wider/narrower it scales with it. Example: new area name that is longer than you've assumed here.